frontend/src/components/Collections/components/Grid/components/Row/CollectionRow/index.tsx

```diff
@@ -12,13 +13,13 @@ import {
   LeftAlignedDetailsCell,
   RightAlignedDetailsCell,
   StyledCell,
-  StyledRow,
+  StyledCollectionRow,
 } from "../common/style";
 import { CollectionTitleText } from "./style";
 
 interface Props {
   id: string;
-  accessType: ACCESS_TYPE;
+  accessType?: ACCESS_TYPE;
```

Comment:
continue optional accessType

Reply:
@hthomas-czi test notif lol No action needed

Reply:
@hthomas-czi again

Reply:
same thread that has mention